ivy/functional/ivy/device.py

```diff
@@ -767,36 +767,25 @@ def tpu_is_available() -> bool:
 
 # noinspection PyShadowingNames
 @handle_exceptions
-def default_device(
-    device: Optional[Union[ivy.Device, ivy.NativeDevice]] = None,
-    /,
-    *,
-    item: Optional[Union[list, tuple, dict, ivy.Array, ivy.NativeArray]] = None,
-    as_native: bool = None,
-) -> Union[ivy.Device, ivy.NativeDevice]:
+def default_device(device=None, *, item=None, as_native=None):
     """
     Return the input device or the default device. If the as_native flag is set, the
     device will be converted to a native device. If the item is provided, the item's
     device is returned. If the device is not provided, the last default device is
     returned. If a default device has not been set, the first gpu is returned if
-    available, otherwise the cpu is returned.
+    available, otherwise the cpu is returned..
 
-    Parameters
-    ----------
-    device
-        The device to be returned or converted.
-    item
-        The item to get the device from.
-    as_native
-        Whether to convert the device to a native device.
+    :param device: The device to be returned or converted.
+    :type device: Union[ivy.Device, ivy.NativeDevice], optional
+    :param item: The item to get the device from.
+    :type item: Union[list, tuple, dict, ivy.Array, ivy.NativeArray], optional
+    :param as_native: Whether to convert the device to a native device.
+    :type as_native: bool, optional
+    :return: Device handle or string.
+    :rtype: Union[ivy.Device, ivy.NativeDevice]
 
-    Returns
-    -------
-    ret
-        Device handle or string.
+    - **Example**
```

Comment:
Why did you change the docstring style? We are using [numpy docstring style](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).

Reply:
I will reverse this but I did it to clarify the hint that I removed in the function args